shared/editor/nodes/Notice.tsx

```diff
@@ -10,6 +11,13 @@ import { MarkdownSerializerState } from "../lib/markdown/serializer";
 import noticesRule from "../rules/notices";
 import Node from "./Node";
 
+export enum NOTICE_TYPES {
+  INFO = "info",
+  SUCCESS = "success",
+  TIP = "tip",
+  WARNING = "warning",
+}
```

Comment:
```suggestion
export enum NoticeTypes {
  Info = "info",
  Success = "success",
  Tip = "tip",
  Warning = "warning",
}
```

Mind if we change the styling? These all caps enums always look so ugly